code/modules/events/grid_check.dm

```diff
@@ -6,10 +6,11 @@
 
 /datum/round_event/grid_check
 	announceWhen = 1
-	startWhen = 1
+	startWhen = 5
```

Comment:
`startWhen = 5` seems to overlap *slightly* with the voice announcement, so you might want to set this to `startWhen = 6`.
Otherwise, 👍.